extensions/job_management/standard/requirements/ogcapi-processes/update/REQ_body.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_ogcapi-processes_update__body]]
+[requirement]
+====
+[%metadata]
+label:: /req/ogcapi-processes/update-body
+part:: The media type `application/ogcapi-processes+json` SHALL be used to indicate that request body contains a job encoded as an <<rc_openeo,OpenEO>>.
```

Comment:
Is that mixed up with extensions/job_management/standard/requirements/ogcapi-processes/create/REQ_body.adoc?

Reply:
Not only mixed up, but probably outdated also, considering https://github.com/opengeospatial/ogcapi-processes/pull/437/files/7b79c502c609ee7ea352499a7fc521ab16551a65#r1790465176